fdbserver/storageserver.actor.cpp

```diff
@@ -4124,6 +4134,322 @@ Key constructMappedKey(KeyValueRef* keyValue, std::vector<Optional<Tuple>>& vec,
 	return mappedKeyTuple.pack();
 }
 
+ACTOR Future<Void> validateRangeAgainstServer(StorageServer* data,
+                                              KeyRange range,
```

Comment:
How big can a `range`'s data become? I think flow has a limit of about 10MB for a packet size to avoid slow tasks. If the range is large, then this function needs some refactor to split the range into smaller ones to compare.

Reply:
The split will happen on the storage server, it will validate via `GetKeyValuesRequest` and check the returned results, and then continue from the last common end key for the next batch.